meetings/2024-07-10.md

```diff
@@ -31,26 +33,104 @@
+* HZ: I don't think anyone disagrees with what you say. Don't see it is a problem but perhaps a missed opportunity. Don't think it's the right time to discuss yet. What gives me hope is lots of people think about this independently. How can we take this to resolution?
+* JZ: If people have opinions about where I should be storing things e.g. temp place for vocabularly then open to suggestions. People should let me know if they think differently. 
+* HZ: So no concrete ask now other than figuring out the storage?
+* JZ: Yes, but for people to be aware of it and if they have ideas to let me know. 
+* RG: Have said this to JZ privately, but he is moving faster than anyone in the community right now with things. 
+* HZ: I agree, also activity pods moving quickly too. Jeff, you are doing a great job and have my support. 
+
+  
+
+### C2C: Project Management
+URL : https://github.com/solid/specification/issues/641
+* HZ what would it take to complete/close this?
+* HZ: Elf started this so hold till next week when he can join. Commonality between this and Jeff's work - how to we store stuff?
+* JZ: Had a practitioners meeting on shapes and created [solid-shapes](https://github.com/solid/shapes) wiki as resource on shapes. Eventually will pull into other catalogue thing. Decided wiki we could all contribute to was best thing.
+
+
+### How to continue to work around user stories
+URL: https://hackmd.io/oqXhIuS4SbmtmuzD0eufeQ
+URL: https://github.com/solid/user-stories
+
+### How can the CG ingest feedback from Solid users?
+* JZ: Suggested resources should be made available but was put down to say CG was about forming specifications and not for us to outreach to communities, publicise things, drawing wider community. This was when CG was reorganising, and so practitioners was formed as part of solid team - that's it's purpose to grow forum, solidproject.org, github things that are managed etc. 
+* HZ: For the record, when I volunteered for co-chair I said my role would be to outreach and grow the community. So do not agree that the CG is about that - I feel it is exactly that.
+* JZ: Go back and read discussion about that. CG has been so focused on spec it has no connection to new users.
+* HZ: I agree. Part of the CG is to received feedback from the community.
+* RG: I will link you to it, but essential push from SC is that because this is a "W3C CG" it has to do work that is to be channelled into Solid WG. So had supported Jeff but then agreed for the charter to happen. That is why Jeff went on to form own practitioners group, which was wise to do. 
```

Comment:
```suggestion
* JZ: Go back and read discussion about that. CG has been so focused on spec, it has no connection to new users.
* HZ: I agree. Part of the CG is to received feedback from the community.
* RG: I will link you to it, but essential push from SC is that because this is a "W3C CG", it has to do work that is to be channelled into Solid WG. So had supported Jeff, but then agreed for the charter to happen. That is why Jeff went on to form own Practitioners group, which was wise to do. 
```